verification_token/migrations/0004_migration.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.15 on 2020-11-12 13:38
+
+from django.db import migrations, models
+from django.utils.timezone import timedelta
+
+
+def recompute_value_for_new_field(apps, schema_editor):
+    VerificationToken = apps.get_model('verification_token', 'VerificationToken')
+    expiring_tokens_qs = VerificationToken.objects.filter(expiration_in_minutes__isnull=False)
+    expiration_minutes = set(
+        expiring_tokens_qs.order_by('expiration_in_minutes').distinct().values_list('expiration_in_minutes', flat=True)
```

Comment:
I think you must write this with

expiring_tokens_qs.order_by('expiration_in_minutes').values('expiration_in_minutes').distinct().values_list('expiration_in_minutes', flat=True)

this maybe will work in PostgreSQL but in the MySQL not

Reply:
Hmm, that seems like an interesting use, if it works. Will try. Otherwise, let's discuss this over phone.

Reply:
FYI, both versions (yours and mine) of the full command use the following SQL query on PostgreSQL:
`SELECT DISTINCT "verification_token_verificationtoken"."expiration_in_minutes" FROM "verification_token_verificationtoken" WHERE "verification_token_verificationtoken"."expiration_in_minutes" IS NOT NULL ORDER BY "verification_token_verificationtoken"."expiration_in_minutes" ASC`
But reading through the docs once again, I have to admit yours should work better (if there are no quirks in other DB adapter implementations - at least none are mentioned in the docs).